gax-grpc/src/main/java/com/google/api/gax/grpc/OperationFuture.java

```diff
@@ -121,58 +127,19 @@ public ResponseT get(long timeout, TimeUnit unit)
     return resultFuture.get();
   }
 
-  /**
```

Comment:
Please put the documentation back! Or make a `{@link}` reference to the overridden method if the doc is present in the parent class.

(This comment is applicable to all methods in this class, especially those with long docs).